kompendium-core/src/test/kotlin/org/leafygreens/kompendium/KompendiumTest.kt

```diff
@@ -486,6 +503,16 @@ internal class KompendiumTest {
     }
   }
 
+  private fun Application.emptyGet() {
+    routing {
+      route("/test/empty") {
+        notarizedGet<Unit, Unit>(emptyTestGetInfo) {
```

Comment:
Hmm strange that this didn't work, I thought I had squashed this bug already 😅 Thanks for fixing